guideml.c

```diff
@@ -2678,15 +2741,15 @@ if(!wb)
   param.bar   = defbar;
   param.retrace = NULL;
 
-	param.colours[0] = 0x000000;
-	param.colours[1] = 0xffffff;
-	param.colours[2] = 0x000000;
-	param.colours[3] = 0x4a8abd;
-	param.colours[4] = 0x000000;
-	param.colours[5] = 0xcfcfcf;
-	param.colours[6] = 0xffffff;
+	param.colours[0] = 0x000000; // text
```

Comment:
Functional change: the default colors are now black on white. The "shine" color becomes red/pink instead of white and the "highlight" color becomes light blue instead of white.

Reply:
The original colours were picked to mimic AmigaGuide. I'm not too fussed about this. A defaults toggle in the GUI might be preferable? At one point I was considering a dark mode option which would also work here. Original/Light/Dark buttons to set the defaults maybe?

Reply:
I see that the colors are set twice, and I changed only the command-line version. So if you use the GUI you would still get the AmigaGuide colors.

Also when using the GUI, these can already be changed using the COLOUR0 to COLOUR6 tooltypes, but there are no matching command-line parameters (which is why I ended up hardcoding my preferred colors).

I will not try to update the GUI myself (no way to test it), I could try to add the COLOUR0 to COLOUR6 parameters in readargs, maybe.

Reply:
I didn't realise they were set twice - this wasn't originally my program and I can't remember what I did when I added the GUI (twenty years or so ago...). Some command line overrides would be fab. I will do some presets in the GUI once it's all merged.

Reply:
Ok, there is now a THEME option with a choice between AmigaGuide, Light and Dark themes. And the color list is in a single place.

Reply:
Fab, thanks.